go-controller/pkg/node/node_ip_handler_linux_test.go

```diff
@@ -180,6 +181,11 @@ var _ = Describe("Node IP Handler tests", func() {
 		dummyMasqIPv6CIDR        = dummyMasqIPv6 + "/125"
 	)
 
+	defer GinkgoRecover()
+	if os.Getenv("NOROOT") == "TRUE" {
+		Skip("Test requires root privileges")
```

Comment:
hmm shouldn't we just use `OnSupportedPlatformsIt` instead of `It` in your below tests?
I know what you have done here works too but since we do have a util specifically added for this...

Reply:
although now I see same-ish method used for eip, rule-manager, route-manager so perhaps we are using both?

Reply:
Its better to use `OnSupportedPlatformsIt`.